infrastructure/pyd/make_object.d

```diff
@@ -445,6 +451,21 @@ T python_to_d(T) (PyObject* o) {
     } else static if (is(T == void)) {
         if (o != cast(PyObject*) Py_None()) could_not_convert!(T)(o);
         return;
+    } else static if (isSlice!T) {
```

Comment:
I think.. here too we are only supporting the case of a numpy array with stride 1.